core/views/browse.py

```diff
@@ -31,7 +31,7 @@
 
 LOGGER = logging.getLogger(__name__)
 
-@cache_page(settings.DEFAULT_TTL_SECONDS)
+@cache_page(settings.DEFAULT_TTL_SECONDS, settings.SHARED_CACHE_MAXAGE_SECONDS)
```

Comment:
Seeing `settings.SHARED_CACHE_MAXAGE_SECONDS` repeated so many times makes me think that `cache_page` should use that by default. I'm wondering whether the same is true for the TTL; there could be an argument against that on the grounds of being explicit but repetition tends to make people miss the few places where something is intentionally different.